apps/language_server/test/server_test.exs

```diff
@@ -235,13 +235,13 @@ defmodule ElixirLS.LanguageServer.ServerTest do
   end
 
   setup context do
-    unless context[:skip_server] do
+    if context[:skip_server] do
```

Comment:
note: unless with an else is flagged by credo for legibility.

Reply:
I don’t think that it’s a code smell